ui/pages/confirmations/components/confirm/info/typed-sign/typed-sign.tsx

```diff
@@ -58,7 +58,7 @@ const TypedSignInfo: React.FC = () => {
         <PermitSimulation tokenDecimals={decimals} />
       )}
       <ConfirmInfoSection>
-        {isPermit && (
+        {isPermit && spender && (
```

Comment:
possible to have a permit without spender ?

may be a malicious one

Reply:
hm so actually, I think we want to split order types from permit types. Order or OrderComponents may not have a spender. 

Orders are conceptually different from Permits as they are used to define trades permits define spending allowances.

In regards to potentially malicious signatures, I think I see what you mean. Allowing the application to break here in case a spender is not found while we categorize this as a permit can be desired. Thanks for the comment! Let me update the code

Reply:
Updated 8dc3f9f